packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -185,28 +176,26 @@ async function executePreview(
     return;
   }
 
-  if (platformSelection.id === PreviewPlatformType.Desktop) {
-    await startServer(true, componentName, startTime);
-    return;
-  }
-
```

Comment:
our Preview command now handles Desktop scenario too, so we can clean up these references here